docs/topics/returns.md

```diff
@@ -111,7 +111,8 @@ fun main() {
 
 Note that the use of local returns in the previous three examples is similar to the use of `continue` in regular loops.
 
-There is no direct equivalent for `break`, but it can be simulated by adding another nesting lambda and non-locally returning from it:
+There is no direct equivalent for `break`, but it can be simulated by adding another nesting lambda and non-locally
+returning from it, as long as the lambda is passed to an [inline function](inline-functions.md):
```

Comment:
> but it can be simulated by adding another nesting lambda and non-locally returning from it:

But the lambda we are adding is outer, not nested

Nested is the `forEach` lambda. Outer is the `run` lambda. The difference from the previous example is the addition of forEach lambda.

---

Please note that if you simply update the sentence to say "by adding another outer lambda", the next part of the sentence becomes incorrect:

> as long as the lambda is passed to an [inline function](inline-functions.md):

Because *the lambda* no longer refers to nested lambda, but it refers to the outer lambda, and the outer lambda doesn't have to be inline. Only the nested lambda has to be.